src/XamlIl.Cecil/CecilMethod.cs

```diff
@@ -80,13 +89,30 @@ class CecilMethod : CecilMethodBase, IXamlIlMethodBuilder
             {
             }
 
+            private CecilMethod(CecilTypeSystem typeSystem, GenericInstanceMethod method)
```

Comment:
I think this might need to be accounted for in Equals.

Reply:
The `FullName` implementation of `GenericInstanceMethod` includes the generic parameters so I think we're good here on the `Equals` implementation.